hardware/openwebnet/bt_openwebnet.h

```diff
@@ -195,6 +195,27 @@ typedef enum {
 	LOAD_CONTROL_WHERE_CONTRL_UNIT = 10
 }_eLoadControlWhere;
 
+/* Where sound diffusion system WHO=22 */
+/*
+|Description	|	Value
+--------------------------------------
+|Source			|	2#sourceID		  |
+|Speaker		|	3#area#point	  |
+|Speaker Area	|	4#area			  |
+|General		|	5#sender_address  |
+|All Source		|	6				  |
+--------------------------------------
+*/
+typedef enum {
```

Comment:
Please remove the typedef as it is not defined.
Could you place this enum in the .cpp file as it is not used in the header (saves compile time)